src/main/java/com/myproject/todo/todo/model/Todo.java

```diff
@@ -0,0 +1,45 @@
+package com.myproject.todo.todo.model;
+
+import java.util.Date;
+
+public class Todo {
+	
+	private Integer id;
+	private Date date; 
+	private String task;
+	public Integer getId() {
+		return id;
+	}
+	public void setId(int id) {
+		this.id = id;
+	}
+	public Date getDate() {
+		return date;
+	}
+	public void setDate(Date date) {
+		this.date = date;
+	}
+	public String getTask() {
+		return task;
+	}
+	public void setTask(String task) {
+		this.task = task;
+	}
+	
+	protected Todo()
+	{
+		
+	}
+	
+	public Todo(int id, Date date, String task) {
```

Comment:
why is this needed when there are setters and an empty constructor?

Reply:
It is kept for instance if a object containing All the data is to be initialized

Reply:
Same thing can be accomplished by an empty constructor and setters. why have both?